assets/src/edit-story/elements/video/test/output.js

```diff
@@ -74,19 +73,8 @@ describe('Video output', () => {
     await expect(<VideoOutput {...props} />).toBeValidAMPStoryElement();
   });
 
-  it('an undefined alt tag in the element should fall back to the resource', async () => {
-    const props = {
-      ...baseProps,
-      element: { ...baseProps.element, alt: undefined },
-    };
-    const output = <VideoOutput {...props} />;
-    await expect(output).toBeValidAMPStoryElement();
-    const outputStr = renderToStaticMarkup(output);
-    await expect(outputStr).toStrictEqual(expect.stringMatching('alt text'));
-  });
-
   it('an empty string alt tag in the element should not fall back to the resource', async () => {
```

Comment:
Pointless test now because the `alt` attribute on the element does not exist anymore. Only `title`

Reply:
Remove the pointless test.